scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -64,6 +75,12 @@ class RunCoverage(
   private val rootDirectory = File(repoRoot).absoluteFile
   private val testFileExemptionTextProto = "scripts/assets/test_file_exemptions"
 
+  companion object {
+    // The minimum coverage percentage threshold for coverage analysis,
+    // The script will fail if the file has less than the minimum specified coverage.
+    const val MIN_COVERAGE_PERCENTAGE = 20
```

Comment:
Where did this number (20) come from? We haven't discussed yet what the default value should be for new code, so I'm curious how you came to this specific value.

Reply:
Sorry for introducing that. I was testing a default minimum threshold for the package files to exit on getting lower coverage results and chose a slightly relevant value based on the issue tracker specifications, which stated: 
>Note that the minimum enforced coverage for now should be something very modest, like 10%. This number will be increased in future changes outside the scope of this project (as the number may not be believable until we understand the limitations of what can be covered by JaCoCo).

However, that was **just an experiment** and will be adjusted according to the final decision. Or may be I can just remove that for now?

Reply:
Removed the const declaration, that will be re-introduced in M2 to check for coverage fail / pass cases.